rpgame/src/main/java/pk/rpgame/view/ExplorationView.java

```diff
@@ -39,12 +39,16 @@ private int getAction() {
   }
 
   public int getRoomChoice(List<Room> nearestRooms) {
-    System.out.println("Możliwe pomieszczenia, do których możesz się udać: " + nearestRooms);
-    System.out.print("Wybierz numer pokoju, do którego chcesz przejść: ");
+    System.out.println("Rooms available to visit: ");
+    for (Room room : nearestRooms) {
+      int i = 1;
+      System.out.println(i + ". " + room.getName());
+    }
```

Comment:
Każdy pokój będzie miec ten sam numer. Jak bohater będzie umiał w klonowanie to może się to rozważy :p

Reply:
Jeszcze inkrementacja i 😄